src/main/java/md2k/mcerebrum/cstress/features/ECGDataQuality.java

```diff
@@ -50,8 +69,22 @@ public ECGDataQuality(DataStreams datastreams, double qualityThreshold) {
         DataPointStream ecg = datastreams.getDataPointStream(StreamConstants.ORG_MD2K_CSTRESS_DATA_ECG);
         DataPointStream ecgQuality = datastreams.getDataPointStream(StreamConstants.ORG_MD2K_CSTRESS_DATA_ECG_QUALITY);
 
-        ECGQualityCalculation ecgComputation = new ECGQualityCalculation(3, 50, 4500, 20, 2, 47);
-        List<DataPoint> quality = ecgComputation.computeQuality(ecg.data, 5000); //0.67
+        this.acceptableOutlierPercent = AUTOSENSE.QUALITY_acceptableOutlierPercent;
+        this.outlierThresholdHigh = AUTOSENSE.QUALITY_outlierThresholdHigh;
+        this.outlierThresholdLow = AUTOSENSE.QUALITY_outlierThresholdLow;
+        this.ecgThresholdBandLoose = AUTOSENSE.QUALITY_ecgThresholdBandLoose;
+        this.bufferLength = AUTOSENSE.QUALITY_bufferLength;
```

Comment:
What is the purpose of a constant 2000 unit buffer?  If this is arbitrary, please use something like ArrayList to handle this without forcing.  See line 87 for an example.